src/WebApi/WebApi.csproj

```diff
@@ -6,43 +6,28 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
-    <!-- as per https://github.com/dotnet/aspnetcore/issues/50836#issuecomment-1819612514  -->
-    <!-- recommendation from msft is to disable AD0001 until first servicing release of .net8   -->
-    <NoWarn>$(NoWarn);AD0001</NoWarn>
+    <UserSecretsId>6893d2b2-792c-4bb6-a737-64aa9cd6e7a6</UserSecretsId>
```

Comment:
User secrets ID needs to be removed

Reply:
https://github.com/SSWConsulting/SSW.CleanArchitecture/pull/235